x-pack/plugins/security_solution/public/timelines/pages/timelines_page.test.tsx

```diff
@@ -72,22 +75,28 @@ describe('TimelinesPageComponent', () => {
     });
 
     test('should show the import timeline button', () => {
-      expect(wrapper.find('[data-test-subj="open-import-data-modal-btn"]').exists()).toEqual(true);
+      expect(wrapper.find('[data-test-subj="timelines-page-open-import-data"]').exists()).toEqual(
+        true
+      );
     });
 
-    test('should show the import timeline modal after user clicking on the button', () => {
-      wrapper.find('[data-test-subj="open-import-data-modal-btn"]').simulate('click');
+    test.skip('should show the import timeline modal after user clicking on the button', () => {
```

Comment:
`skip` forgotten?

Reply:
it wasn't actually forgotten, the test was failing because I made changes that were rendering things not accessible by the `shallow` function from `enzyme`. I didn't want to change this file too much because I felt that it was outside of the scope of this PR...
I actually tried to get rid of `enzyme` entirely and completely re-write the file, but the `StatefulTimelineComponent` is EXTREMELY (too) complex and I couldn't get anything to work as they were too many things to mock.

I decided to rewrite the entire test file, but still using `enzyme`, hopefully this is acceptable!

Future work should refactor this page as the logic there is overly complex. Like why are we passing down the `isImportDataModalOpen` and `setImportDataModal` via props to the `StatefulTimelineComponent` instead of having the modal open at the top level??

Reply:
Okay, yes migrating from enzyme to react test lib is hard sometimes, especially if it is using `shallow` rendering. No worries, I don't think we should spend too much time migrating tests that are already working, that's acceptable for me.